sandbox-tools/merge-repos/src/syncReposToStaging.ts

```diff
@@ -265,6 +207,31 @@ function getFileStatus(status: StatusResult, name: string): FileStatusResult {
     return null;
 }
 
+function logAppendMessage(commitMessage: string, fileStatus :FileStatusResult, message: string) {
```

Comment:
moved out of the `resolveConflictsToTheirs` function so other functions in this script could also use it.